pkg/api/org_invite.go

```diff
@@ -39,7 +39,7 @@ func AddOrgInvite(c *m.ReqContext, inviteDto dtos.AddInviteForm) Response {
 		}
 
 		if setting.DisableLoginForm {
-			return Error(401, "User could not be found", nil)
+			return Error(404, "User could not be found", nil)
```

Comment:
Not sure this is correct either.
If login is disabled it should not be possible to invite others. If people are invited I think Grafana should return 400. What do you think?

Reply:
If so, the code should be moved outside the if statement.

Reply:
@bergquist  Thanks for the insight. Wasn't aware of the use case where you can't invite others when login is disabled".

So in this case I have moved the check for `setting.DisableLoginForm` to the top of the function. That is, if it is `disabled` then don't bother executing the remaining code path. Let me know if I am right.

Then in this context, yes `400` would make sense as the `http request` will deem as invalid because you can't do it. Next closest choice I think would be `501` not implemented but that is not really the case, is there it is just not `enabled` so probably 400.